src/pages/Profile/result.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Card } from 'antd';
+function MyResult(props) {
+    return (
+
+        <Card>
+            <div className="card-body">
+                <h1 >Save lives, stay home!</h1>
+                <img src={props.src} className="avatar" alt='avatar' />
+                <h3>{props.name} {props.emoji}</h3>
+                <p>{props.address}</p>
+                <p>{props.description}</p>
+            </div>
+        </Card>
+    )
+}
+export default MyResult;
```

Comment:
ơ cái VS ko tự thêm cuối dòng cho file à . xem lại cái prettier 
https://prettier.io/docs/en/options.html#end-of-line
thêm cái này vào file .prettierrc